tests/game.test.js

```diff
@@ -0,0 +1,9 @@
+import Game from '../src/modules/game';
```

Comment:
**[ REQUIRED ]:** Since the whole scope of the project is based around TDD, I must make the observation that you did not fully test your factory methods, in general all I can see is the call for a single method. Testing should be more through, like you should be testing for the isSunk() method in the ship factory, you should be testing the allSunk() method, etc... not just a singular function of every factory. Since that is the scope of the project as stated [HERE](https://www.theodinproject.com/courses/javascript/lessons/battleship): 
![image](https://user-images.githubusercontent.com/46115725/70769315-9a246900-1d2e-11ea-9137-fc3810e5f7f4.png)

**I M P O R T A N T** It is important that your tests are meaningful and that they all pass, currently only one of your tests is passing as you can see: 
![image](https://user-images.githubusercontent.com/46115725/70769375-e1125e80-1d2e-11ea-93af-e02cb884db69.png)

- I should also ask you to extend the reach of your tests to cover the methods included in your factory.